### ISSUE TITLE
Option to include README in jar

### ISSUE THREAD

**Issue by stevedlawrence** [apache/daffodil-sbt#7]
Since we often distribute schemas as jars, it would be useful to include the README as well, since this file usually includes helpful things like usage and license information, which isn't necessarily in the schema files.

**Comment by stevedlawrence** [comment]
We should also include LICENSE/NOTICE files if they exist.

**Comment by olabusayoT** [comment]
I wonder if we can also add build.sbt, since all of our schema projects rely on it.

**Comment by mbeckerle** [comment]
If one adds a src/main/resources/doc directory with files in it, are those going to end up in a doc subdir in the jar, or does some special provision need to be made to get that to happen?

**Comment by stevedlawrence** [comment]
Yep, anything in src/main/resources ends up in the root of the jar, including any subdirectories. Nothing special needs to be done.

**Comment by mbeckerle** [comment]
So perhaps all we need is guidelines that the README.md is for us, and user doc, rel-notes, instructions, whatever we want to include is just referenced from the README but lives under src/main/resources/doc ?

**Comment by olabusayoT** [comment]
Closing as wont-fix. We recommend adding mappings to the build.sbt as below

```
    Seq(packageBin, packageSrc, packageDoc).map { task =>
      Compile / task / mappings ++= Seq("COPYRIGHT.txt", "LICENSE.txt", "README.md")
        .map{ name => (baseDirectory.value / name) -> name }
    }
```
